l3kernel/l3fp-aux.dtx

```diff
@@ -245,12 +245,27 @@
 %    \end{macrocode}
 % \end{macro}
 %
-% \begin{macro}{\s_@@_mark, \s_@@_stop}
+% \begin{variable}{\s_@@_expr_mark, \s_@@_expr_stop}
 %   Aliases of \cs{tex_relax:D}, used to terminate expressions.
 %    \begin{macrocode}
+\scan_new:N \s_@@_expr_mark
+\scan_new:N \s_@@_expr_stop
+%    \end{macrocode}
+% \end{variable}
+%
+% \begin{variable}{\s_@@_mark, \s_@@_stop}
+%   Generic scan marks used throughout the module.
+%    \begin{macrocode}
 \scan_new:N \s_@@_mark
 \scan_new:N \s_@@_stop
 %    \end{macrocode}
+% \end{variable}
```

Comment:
Assuming you stick with having scan marks and quarks specific to each module, I agree that distinguishing the `\s_@@_expr_` ones from the `\s_@@_` ones is useful in `l3fp`.

Reply:
Not only useful, it was necessary: the code reads the name of the scan
mark, and will break badly if it's an unexpected name.